app/src/main/java/org/oppia/app/splash/SplashFragment.kt

```diff
@@ -0,0 +1,45 @@
+package org.oppia.app.splash
+
+import android.content.Intent
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import org.oppia.app.HomeActivity
+import org.oppia.app.R
+
+/*
+* The SplashFragments navigates to Home page once
+* the app is finished loading completely
+* */
+
```

Comment:
Nit: Javadocs should not have a blank space between them and the class/method/field that they're documenting.